client/task-list/tasks/payments/RemotePayments/components/PaymentMethodList/PaymentMethodList.js

```diff
@@ -87,7 +88,9 @@ export const PaymentMethodList = ( {
 							<PaymentAction
 								manageUrl={ manageUrl }
 								methodKey={ key }
-								hasSetup={ !! fields.length }
+								hasSetup={
+									!! plugins.length || !! fields.length
```

Comment:
Something like `Boolean(plugins.length || fields.length)` might be clearer at this point.